src/app/components/Footer/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledFooter = styled.footer`
+  background-color: #404040;
+  font-family: ReithSans, Arial, Helvetica, freesans, sans-serif;
+  padding: 16px;
+`;
+
+const StyledParagraph = styled.p`
+  color: #fff;
+`;
+
+const Footer = () => (
+  <StyledFooter>
```

Comment:
@jp-bbc can we add role contentinfo please?

Reply:
Yep, Done